devicetypes/heltun/heltun-tps05-switch.src/heltun-tps05-switch.groovy

```diff
@@ -0,0 +1,723 @@
+                35: "3rd & 5th Relay",
+                45: "4th & 5th Relay"
+        ], paramNum: 102, size: 1, default: "0", type: "enum"], 
+
+    [title: "Energy Consumption Meter Consecutive Report Interval", description: "When the device is connected to the gateway, it periodically sends reports from its energy consumption sensor even if there is no change in the value. " +
+    "This parameter defines the interval between consecutive reports of real time and cumulative energy consumption data to the gateway",
+     name: "Selected Energy Report Interval in minutes", paramNum: 141, size: 1, default: 10, type: "number", min: 1 , max: 120, unit: "min"],
+
+    [title: "Control Energy Meter Report", description: "This Parameter determines if the change in the energy meter will result in a report being sent to the gateway. " +
+    "Note: When the device is turning ON, the consumption data will be sent to the gateway once, even if the report is disabled.",
+     name: "Sending Energy Meter Reports", options: [
+                0: "Disabled",
+                1: "Enabled"
+        ], paramNum: 142, size: 1, default: "1", type: "enum"],
+
+    [title: "Sensors Consecutive Report Interval", description: "When the device is connected to the gateway, it periodically sends to the gateway reports from its external " +
+    "NTC temperature sensor even if there are not changes in the values. This Parameter defines the interval between consecutive reports",
+     name: "Selected Energy Report Interval in minutes", paramNum: 143, size: 1, default: 10, type: "number", min: 1 , max: 120, unit: "min"],
+
+    [title: "Air & Floor Temperature Sensors Report Threshold", description: "This Parameter determines the change in temperature level (in °C) resulting in temperature sensors " +
+    "report being sent to the gateway. The value of this Parameter should be x10 for °C, e.g. for 0.4°C use value 4. Use the value 0 if there is a need to stop sending the reports.",
+     name: "Selected Temperature Threshold in °Cx10", paramNum: 144, size: 1, default: 2, type: "number", min: 0 , max: 100, unit: " °Cx10"],
+
+    [title: "Humidity Sensor Report Threshold", description: "This Parameter determines the change in humidity level in % resulting in humidity sensors " +
+    "report being sent to the gateway. Use the value 0 if there is a need to stop sending the reports.",
+     name: "Selected Humidity Threshold in %", paramNum: 145, size: 1, default: 2, type: "number", min: 0 , max: 25, unit: "%"],
+
+    [title: "Light Sensor Report Threshold", description: "This Parameter determines the change in the ambient environment illuminance level resulting in a light sensors report " +
+    "being sent to the gateway. From 10% to 99% can be selected. Use the value 0 if there is a need to stop sending the reports.",
+     name: "Selected Light Sensor Threshold in %", paramNum: 146, size: 1, default: 50, type: "number", min: 0 , max: 99, unit: "%"]
+
+]}
```

Comment:
now You need to replace 4 spaces with 1 tab

Reply:
@KKlimczukS my fault.
Fixed